style.css

```diff
@@ -0,0 +1,235 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+.head {
+  color: #6070ff;
+  padding: 24px;
+  text-align: center;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-top: 14px;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  width: 76px;
+  height: 20px;
+}
+
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible.
`The typo(font,color), spacing(margin, padding) need to be similar with the ones provided in the Figma template.`Kindly correct that.
Your design :point_down: 

![Screenshot from 2022-10-27 15-06-51](https://user-images.githubusercontent.com/97100091/198293972-1af640f6-190d-446d-851d-87ca5e467f36.png)

Figma design :point_down: 

![image](https://user-images.githubusercontent.com/97100091/198294385-32c1f736-5ab0-4875-b9d6-5837f4876b9f.png)